### ISSUE TITLE
cifar10 parameter config

### ISSUE THREAD

**Issue by Evil-Wideplain** [anshulbshah/MMCL#1]
I want to use your method to test it on cifar10 datasets, but the accuracy not very satisfy, can you share your parameter config about train on cifar10?
Waiting for your reply！

**Comment by Evil-Wideplain** [comment]
add:
     Did your each epoch loss was negative when training?

**Comment by anshulbshah** [comment]
Thanks for your interest in our work! 

- The code was used for ImageNet training. 
- Training on datasets like CIFAR-100 requires changes in the first layer of the model due to small size of images (for example https://github.com/joshr17/HCL/blob/main/image/model.py)
- For experiments on CIFAR100, we implement our approach on HCL's framework (https://github.com/joshr17)
- The loss can be negative due to the way it is defined 

The repo is now updated to include some scripts for running CIFAR100 experiments (Please check https://github.com/anshulbshah/MMCL/tree/main/CIFAR100). We did not experiment with pre-training on CIFAR10. You can modify `utils.py` to run experiments on CIFAR-10 (This might be helpful https://github.com/joshr17/HCL/blob/main/image/utils.py).

**Comment by Evil-Wideplain** [comment]
Thanks for your replying, I will try it as you said again.
